src/assets/svg/booster.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">

```

Comment:
You should properly remove the width and height attributes so the parent element controls the size of the icon. This is how all other icons are implemented. (required)

Reply:
👍